/arte without arguments fails on bukkit and it's forks

I'm trying to use paper 1.21.1, and it's giving the following error

[04:39:24 ERROR]: Command exception: /arte
org.bukkit.command.CommandException: Unhandled exception executing command 'arte' in plugin arte v0.4.4-SNAPSHOT
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.command.brigadier.bukkit.BukkitCommandNode$BukkitBrigCommand.run(BukkitCommandNode.java:91) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.3.10.jar:?]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:31) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:5) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:103) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:443) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.Commands.performCommand(Commands.java:350) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.Commands.performCommand(Commands.java:337) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.commands.Commands.performCommand(Commands.java:332) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performUnsignedChatCommand(ServerGamePacketListenerImpl.java:2218) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$11(ServerGamePacketListenerImpl.java:2192) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1537) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:201) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1514) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1507) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1466) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1473) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1318) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:330) ~[paper-1.21.1.jar:1.21.1-74-971a7a5]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
        at arte-bukkit-0.4.4-SNAPSHOT.jar/io.shardmc.arte.bukkit.command.ArteCommand.onCommand(ArteCommand.java:37) ~[arte-bukkit-0.4.4-SNAPSHOT.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        ... 27 more

--- a/bukkit/src/main/java/su/shardmc/arte/bukkit/command/ArteCommand.java
+++ b/bukkit/src/main/java/su/shardmc/arte/bukkit/command/ArteCommand.java
@@ -34,36 +34,35 @@ public class ArteCommand implements TabExecutor {
         if (!sender.hasPermission("arte.command"))
             return true;
 
-        if (args[0].equals("reload")) {
-            this.arte.config().reload();
-
-            new Thread(() -> {
-                this.arte.getPackManager().reload();
-
-                scheduler.runTask(this.arte, () -> {
-                    sender.sendMessage("[Arte] Done!");
-
-                    for (Player player : this.arte.getServer().getOnlinePlayers()) {
-                        this.arte.getPackManager().apply(player);
-                    }
-                });
-            }).start();
+        if (args.length == 0) {
+            sender.sendMessage("[Arte] No arguments given!");
+            return true;
         }
 
-        if (args[0].equals("clean")) {
-            new Thread(() -> {
+        new Thread(switch (args[0]) {
+            case "reload" -> {
+                this.arte.config().reload();
+                yield (() -> {
+                    this.arte.getPackManager().reload();
+
+                    scheduler.runTask(this.arte, () -> {
+                        sender.sendMessage("[Arte] Done!");
+
+                        for (Player player : this.arte.getServer().getOnlinePlayers())
+                            this.arte.getPackManager().apply(player);
+                    });
+                });
+            }
+            case "clean" -> (() -> {
                 try {
                     this.arte.getPackManager().getZipper().clean();
-
-                    scheduler.runTask(this.arte, () ->
-                            sender.sendMessage("[Arte] Done!")
-                    );
+                    scheduler.runTask(this.arte, () -> sender.sendMessage("[Arte] Done!"));
                 } catch (IOException e) {
                     throw new RuntimeException(e);
                 }
-            }).start();
-        }
-
+            });
+            default -> (() -> sender.sendMessage("[Arte] Unexpected argument: " + args[0]));
+        }).start();
         return true;
     }
 

